smileCounter.py

```diff
@@ -22,11 +33,38 @@
     for (x, y, w, h) in face:
         img = cv2.rectangle(frame, (x,y), (x+w, y+h), (0, 0, 255), 3)
         # Detect the Smile
-        smile = smile_cascade.detectMultiScale(gray, scaleFactor=1.8, minNeighbors=20)
+        smile = smile_cascade.detectMultiScale(gray, scaleFactor=1.5, minNeighbors=10)
+        # Increase smile counter by number of detected smiles
+        if len(smile) > 0:
+            if not smile_active:
+                current_time = time.time()
+                if current_time - last_smile_time > TIME_TO_START_COUNTING:
+                    smiles_detected += 1
+                    smile_active = True
+        else:
+            if smile_active:
+                last_smile_time = time.time()
+                smile_active = False
+
         # Draw the rectangle around detected smile
         for x, y, w, h in smile:
             img = cv2.rectangle(frame, (x, y), (x+w, y+h), (255, 0, 0), 3)
 
+        # Get img size
+        h, w, c = frame.shape
+        # Set new img position depending on img size
+        textPosition = (round(w/4), round(h/8))
+        # Text to show
+        textToShow = "Wykryto usmiechow: " + str(smiles_detected)
+        # Add smile count
+        cv2.putText(img, textToShow,
+                    textPosition,
+                    font,
+                    fontScale,
+                    fontColor,
+                    thickness,
+                    lineType)
+
     # Display output
     cv2.imshow('smile detect', frame)
```

Comment:
As you can see, you added messages in Polish and somewhere in the code where you didn't pay attention, the message in English remained.

### ✔️ You don't have to change anything here